assets/component-slider.css

```diff
@@ -160,8 +176,8 @@ slider-component.slider-component-full-width {
     scroll-margin-left: 1.5rem;
   }
 
-  .slider-component-full-width .slider--desktop .slider__slide:last-child {
-    margin-right: 1.5rem;
+  .slider-component-full-width .slider--desktop:after {
+    padding-left: 1.5rem;
```

Comment:
Is there any reason we couldn't use either margin OR padding for this? Using both makes me think they serve different purposes. Do they? I'd slightly prefer if we could just account for the 1.5rem in the same calculation as the negative spacing offset rather than relying on one to counteract the other.

Reply:
So the thing by adding a pseudo `:after` element is that it will add some spacing using the gap like if it was another `li` element. I need the `padding` value to be positive in order for that pseudo element to be showing.

Now due to the `gap` value coming up, I want to cancel it by adding a negative value into the mix. I'm using the padding to set the value of whatever we use for `.page-width` and the margin to cancel whatever the gap value is. 
I can't set a negative value on padding so I can't use that property alone. And margin doesn't get the pseudo element to come up visually so I can't use that alone either. 
Seems to be the best way to go about it from what I tested 🤔